providers/InertiaProvider/InertiaProvider.ts

```diff
@@ -174,7 +174,7 @@ export default class InertiaProvider {
     Response.macro(
       'redirect',
       function (path?: string, forwardQueryString: boolean = false, statusCode = 302): RedirectContract | void {
-        const isInertia = this.request.rawHeaders.includes(HEADERS.INERTIA_HEADER);
+        const isInertia = this.request.headers[HEADERS.INERTIA_HEADER.toLowerCase()];
```

Comment:
I think that we can just change the `HEADERS` enum to be all lowercase intead of converting it here